### ISSUE TITLE
make include/oqs useful to cp -r in or out, or better solution with make install

### ISSUE THREAD

**Issue by listcoin** [open-quantum-safe/liboqs#117]
perhaps i didn't configure liboqs with the correct options, but my makefile links a bunch of the headers into include/oqs, but they're soft links and thus useless when naively copying them into some place more useful (like /usr/local/include/oqs).  also, not all the headers are present, so not all functionality of liboqs is exposed...

perhaps there's a better solution, have automake take care of copying all the header files into /usr/local/include when doing the make install:
http://stackoverflow.com/questions/32148172/move-all-header-files-to-usr-local-include-target
this option seems appropriate, as it wouldn't require any extra steps on the end users part to start working with the library in another C/C++ project.

here are the changes to the Makefile.am which do this, but you may want to add/modify.  (note that it puts it in /usr/local/include/liboqs, though, there should be a way to do it in include/oqs if desired.)

```
pkginclude_HEADERS = src/common/common.h \
                     src/crypto/aes/aes.h \
                     src/crypto/sha3/sha3.h \
                     src/kex/kex.h \
                     src/crypto/rand/rand.h \
                     src/crypto/rand_urandom_chacha20/rand_urandom_chacha20.h \
                     src/crypto/rand_urandom_aesctr/rand_urandom_aesctr.h \
                     src/kex_rlwe_newhope/kex_rlwe_newhope.h \
                     src/kex_rlwe_msrln16/kex_rlwe_msrln16.h \
                     src/kex_lwe_frodo/kex_lwe_frodo.h \
                     src/kex_rlwe_bcns15/kex_rlwe_bcns15.h \
                     src/kex_sidh_cln16/kex_sidh_cln16.h

if USE_MCBITS
pkginclude_HEADERS += src/kex_code_mcbits/kex_code_mcbits.h
endif
if USE_NTRU
pkginclude_HEADERS += src/kex_ntru/kex_ntru.h
endif
if USE_SIDH_IQC
pkginclude_HEADERS += src/kex_sidh_iqc_ref/kex_sidh_iqc_ref.h
endif

```

**Comment by tlepoint** [comment]
@smashra Could we do this?

**Comment by smashra** [comment]
Yes we can, but its not done like that because we don't do

make install.

Shravan

**Comment by lowagner** [comment]
why not?  make install is pretty standard...
(forgive my ignorance!)

**Comment by smashra** [comment]
The point is not that.
We don't.
But that doesn't mean it cannot be done.

**Comment by tlepoint** [comment]
We probably should change the current behavior then :)

**Comment by dstebila** [comment]
Fixed by #157
